.github/workflows/integration-test-components.yml

```diff
@@ -0,0 +1,17 @@
+name: Cypress Tests
+on: push
+
+jobs:
+  cypress-run:
+    name: Test components with Cypress
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: npm run build
+          start: npm start
```

Comment:
I looked into the [failed run](https://github.com/yokoso-capstone/yokoso/runs/4086370827?check_suite_focus=true) by running the action locally (using this [cool tool](https://github.com/nektos/act)) and turned out to be an issue with mismatched npm versions. So the fix is to keep everything consistent with the package-lock.json version and build env by adding the following steps at the beginning, like how the other workflows include: 

```yml
      - uses: actions/checkout@v2
      - name: Setup node
        uses: actions/setup-node@v2
        with:
          node-version: "12"
      - name: Install npm v7
        run: npm install -g npm@7
```

So the whole thing would look like:

```yml
name: Cypress Tests
on: push

jobs:
  cypress-run:
    name: Test components with Cypress
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Setup node
        uses: actions/setup-node@v2
        with:
          node-version: "12"
      - name: Install npm v7
        run: npm install -g npm@7
      # Install NPM dependencies, cache them correctly
      # and run all Cypress tests
      - name: Cypress run
        uses: cypress-io/github-action@v2
        with:
          build: npm run build
          start: npm start
```

Note: I ran it locally using a minimal Docker container where I had to manually install some [Cypress dependencies](https://docs.cypress.io/guides/continuous-integration/introduction#Dependencies). The workflow might have to explicitly install those if the GitHub Actions or `cypress-io/github-action` workflow environment doesn't include them.